packages/core/CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Support to import custom `SideBar` component in `app/` folder that replaces sidebar in Mechanic UI.
+- New design function export `ExtraUi`, expected to be a React component that's added to the bottom section of sidebar, on top of default toggles and buttons.
+- Support for SVG and PNG exports for SVG based engines like `engine-react` and `engine-svg`.
+- New `hideFeedback` design function setting. When true, Mechanic's feedback button is hidden. Defaults to false.
+- New `hideNavigation` design function setting. When true, the navigation input that let's users select a design function is hidden. Defaults to false.
+- New `hidePresets` design function setting. When true, preset selection input is hidden. Defaults to false.
```

Comment:
just to make sure... the preset's selector is still hidden unless there is a `presets` exports, right?

Reply:
Yes!